liquibase-standard/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1513,7 +1501,11 @@ protected void doMigration() throws Exception {
                 liquibase.clearCheckSums();
                 return;
             } else if (COMMANDS.CALCULATE_CHECKSUM.equalsIgnoreCase(command)) {
-                liquibase.calculateCheckSum(commandParams.iterator().next());
+                liquibase.calculateCheckSum(
```

Comment:
I think here we should better perform the `calculateCheckSum` call by the command scope as below to avoid keep using the deprecated method and just leave the **Liquibase** class methods for those who use liquibase as a library: 

```
CommandResults commandResults = new CommandScope("calculateChecksum")
                .addArgumentValue(DbUrlConnectionCommandStep.DATABASE_ARG, database)
                .addArgumentValue(CalculateChecksumCommandStep.CHANGESET_PATH_ARG, changeSetPath)
                .addArgumentValue(CalculateChecksumCommandStep.CHANGESET_ID_ARG, changeSetId)
                .addArgumentValue(CalculateChecksumCommandStep.CHANGESET_AUTHOR_ARG, changeSetAuthor)
                .addArgumentValue(CalculateChecksumCommandStep.CHANGELOG_FILE_ARG, this.changeLogFile)
                .execute();
return commandResults.getResult(CalculateChecksumCommandStep.CHECKSUM_RESULT);
```

Reply:
Added the command scope call